osutil/disks/disks_linux.go

```diff
@@ -144,11 +144,24 @@ func diskFromUdevProps(deviceIdentifier string, deviceIDType string, props map[s
 	// create the full path by pre-pending /sys, since udev doesn't include /sys
 	devpath = filepath.Join(dirs.SysfsDir, devpath)
 
+	// TODO: this doesn't seem to work for /dev/mapper/loop1p1 devices like we
```

Comment:
Out of curiosity, how does the `/sys` tree looks like in a scenario like with the mapper where this does not work?

Reply:
see my response to @bboozzoo I think this arose from my usage of kpartx instead of losetup manually, I guess kpartx is doing things slightly differently than losetup is and losetup is doing the right thing so this TODO can go away for the spread test